Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1730,6 +1730,178 @@ end
 				},
 			},
 		},
+		cColor =
+		{
+			Desc = [[
+				Encapsulates a RGB color, e.g. for armor.
```

Comment:
I think the range of the RGB components should be mentioned somewhere - is it 0 - 255 each, or 0.0 - 1.0 each?